lazycoder1412/3SumSmaller.cpp

```diff
@@ -0,0 +1,18 @@
+class Solution {
+public:
+    int threeSumSmaller(vector<int>& nums, int target) {
```

Comment:
If you are accepting a variables by reference and you don't plan on modifying this variable, always declare it as `const` - `threeSumSmaller(const vector<int> &nums, ...)`

Reply:
Yes but we have to point this otherwise this fact cannot be learned.